resin/base_request.py

```diff
@@ -42,7 +42,9 @@ def __set_authorization(self, headers):
             {'Authorization': 'Bearer {:s}'.format(self.settings.get(TOKEN_KEY))})
 
     def __get(self, url, headers, data=None, stream=None):
-        return requests.get(url, headers=headers, timeout=self.timeout)
+        if not stream:
+            return requests.get(url, headers=headers, timeout=self.timeout)
+        return requests.get(url, headers=headers, stream=stream)
```

Comment:
Why don't you always pass stream to `requests`? No need for the if statement

Reply:
yeah, that's right, updated!